.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Run tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: osp
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Installing dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Creating Zulip API key
+      run: echo ${{ secrets.ZULIP_API_KEY }} >> download
+    - name: Migrating Test Database
+      run: python manage.py migrate
+    - name: Run tests
+      run: python manage.py test --keepdb
```

Comment:
Add a blank line after each step to make it more readable

Reply:
Sure!